dockerfiles/e2e-saas/test_patches/CheApiRequestHandler.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/tests/e2e/utils/requestHandlers/CheApiRequestHandler.ts b/tests/e2e/utils/requestHandlers/CheApiRequestHandler.ts
```

Comment:
We should get rid of this path in the upcoming upstream releases
@Katka92 @ScrewTSW could you please create an issue for that and raise with @rhopp during prioritization / planning

Reply:
It will be part of https://github.com/eclipse/che/issues/16456.